goaviatrix/aws_tgw_vpc_attachment.go

```diff
@@ -28,32 +30,76 @@ type DomainListResp struct {
 func (c *Client) CreateAwsTgwVpcAttachment(awsTgwVpcAttachment *AwsTgwVpcAttachment) error {
```

Comment:
Not sure if this is the right way.
 goaviatrix should have two API's for TGQ VPC attachment and firenet.
 and terraform should call the right API depending on the IsFirewallSecurityDomain output